server.js

```diff
@@ -76,6 +77,12 @@ app.get('/config.js', (req, res) => {
   }
 });
 
+// new metrics endpoint for Walkthrough applciation.
```

Comment:
```suggestion
// new metrics endpoint for Walkthrough application.
```
(if leaving the comment)